src/BingoGame.Indexer.CA/GraphQL/GetBingoDto.cs

```diff
@@ -0,0 +1,15 @@
+using GraphQL;
+using Nest;
+using BingoGame.Indexer.CA.GraphQL;
+using Volo.Abp.Application.Dtos;
+
+namespace BingoGame.Indexer.CA.Entities;
```

Comment:
The namespace doesn't align with the folder name. Please fix it.

Reply:
noted, changed Entities to GraphQL